tools/oeutil/host/generate_evidence.cpp

```diff
@@ -1212,13 +1212,20 @@ oe_result_t generate_oe_evidence(
         const oe_claim_t* claim;
         oe_result_t _verify_evidence_result;
         std::vector<oe_policy_t> policies = {};
+        uint8_t* endorsements_buffer = endorsements;
+        size_t endorsements_buffer_size = endorsements_size;
         if (_parameters.baseline != NULL)
         {
             oe_policy_t policy = {
                 OE_POLICY_ENDORSEMENTS_BASELINE,
                 (void*)_parameters.baseline,
                 strlen(_parameters.baseline) + 1};
             policies.emplace_back(policy);
+
+            // Endorsements need to be recreated as endorsements baseline is
+            // specified
+            endorsements_buffer = NULL;
```

Comment:
nullptr

Reply:
keep NULL to be consistent with other parts, but we should have a cleanup